test/unit/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -38,7 +38,7 @@ class WithoutRenderer < JsonApiRendererTest
       require 'active_record'
       require 'support/rails5_shims'
       require 'active_model_serializers'
-      require 'fixtures/poro'
+      require_relative '../../support/fixtures/poro'
```

Comment:
when tests run they include the `test` directory in the search path, so `require 'support/fixtures/poro'` is what you want